config/global.ini.php

```diff
@@ -910,12 +910,15 @@
 [Deletelogs]
 ; delete_logs_enable - enable (1) or disable (0) delete log feature. Make sure that all archives for the given period have been processed (setup a cronjob!),
 ; otherwise you may lose tracking data.
-; delete_logs_schedule_lowest_interval - lowest possible interval between two table deletes (in days, 1|7|30). Default: 7.
+; delete_logs_schedule_lowest_interval - lowest possible interval between two table deletes, for table log_visit (in days, 1|7|30). Default: 7.
```

Comment:
this would affect all log tables, not just log_visit

Reply:
Not sure why it was resolved. Should we mention it is applied to all `log_*` tables meaning all tables that start with `log_`?

Reply:
This is fixed in the latest commit